server/controllers/allMentors.js

```diff
@@ -1,28 +1,41 @@
 
-import allusers from '../models/users';
+import pool from '../configurations/db-config';
 
-const mentorsDisplay = (req,res) => {
-     const check = [];
-    allusers.forEach(mentor => {
-        if(mentor.is_mentor === true){
-            
-            let mentorData ={
-                firstName: mentor.firstName ,
-                lastName: mentor.lastName,
-                email: mentor.email, 
-                address: mentor.address,
-                bio: mentor.bio,
-                occupation: mentor.occupation,
-                expertise: mentor.expertise,
+class mentors {
+  async mentorsDisplay(req, res) {
+    try {
+      const mentors = 'SELECT * FROM USERS';
```

Comment:
Why don't you just query only mentors since you have the is_mentor column in the database? 
Imagine you had 100000 users and you only have 2 mentors, Your implementation won't be efficient.